common/util.go

```diff
@@ -63,16 +62,12 @@ func RemoveContents(directory string) error {
 }
 
 // ExtractRepository gets the project name from its URL in the form [username]/[project]
-func ExtractRepository(URL string) (string, error) {
-	const defaultName = "$YOUR_REPOSITORY"
-	r, err := regexp.Compile(`.com(/|:)(.+/.+)`)
+func ExtractRepository(URL string) string {
+	const DefaultName = "$YOUR_REPOSITORY"
```

Comment:
just wondering, is this a Go naming convention we should be adhering to (`VarName` vs `varName` for `const`)?

Reply:
I was wondering too. All the GoDoc mentions is capitalize for exports, but we're inside a function here so I guess it doesn't matter 🤷‍♂️

Reply:
https://stackoverflow.com/questions/22688906/go-naming-conventions-for-const

camelCase looks fine. Otherwise capitalize for exports.

Reply:
yeah I think capital-first is used to denote exports, should stick to lower-case for everything else regardless of const or not